.github/workflows/tests.yml

```diff
@@ -0,0 +1,27 @@
+permissions:
+  contents: read
+on:
+  push:
+
+# See https://docs.github.com/en/actions/using-jobs/using-concurrency
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+name: Tests
+jobs:
+  basic:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+      - name: Install stable
+        uses: dtolnay/rust-toolchain@1.85.0
```

Comment:
would make sense to pin latest stable?

Reply:
I think it does. Changed in https://github.com/newrelic/rust-licenses-noticer/pull/1/commits/9401fce7a4cf1fe04de48bc9185fe3dc60c9129f!